v20.2/known-limitations.md

```diff
@@ -8,78 +8,64 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
-See: https://github.com/cockroachdb/cockroach/issues/46414
-~~~
+CockroachDB does not currently support [`IMPORT`s](import.html) into tables with partial indexes.
 
-[Tracking Github Issue](https://github.com/cockroachdb/cockroach/issues/46414)
+To work around this limitation:
 
-### Column name from an outer column inside a subquery differs from PostgreSQL
+1. Drop any partial indexes defined on the table.
+2. Perform the `IMPORT`.
+3. Recreate the partial indexes.
 
-CockroachDB returns the column name from an outer column inside a subquery as `?column?`, unlike PostgreSQL. For example:
+If you are [performing an `IMPORT` of a `PGDUMP`](migrate-from-postgres.html) with partial indexes:
 
-~~~ sql
-> SELECT (SELECT t.*) FROM (VALUES (1)) t(x);
-~~~
+1. Drop the partial indexes on the PostgreSQL server.
+2. Recreate the `PGDUMP`.
+3. `IMPORT` the `PGDUMP`.
+4. Add partial indexes on the CockroachDB server.
 
-CockroachDB:
+[Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/50225)
 
-~~~
-  ?column?
-------------
-         1
-~~~
+### Spatial support limitations
 
-PostgreSQL:
+CockroachDB supports efficiently storing and querying [spatial data](spatial-data.html), with the following limitations:
 
-~~~
- x
----
- 1
-~~~
+- Not all [PostGIS spatial functions](https://postgis.net/docs/reference.html) are supported.
 
-[Tracking Github Issue](https://github.com/cockroachdb/cockroach/issues/46563)
+    [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/49203)
 
-### Privileges required to access to certain virtual tables differs from PostgreSQL
+- The `AddGeometryColumn` [spatial function](functions-and-operators.html#spatial-functions) only allows constant arguments.
 
-Access to certain virtual tables in the `pg_catalog` and `information_schema` schemas require more privileges than in PostgreSQL. For example, in CockroachDB, access to `pg_catalog.pg_types` requires the `SELECT` privilege on the current database, whereas this privilege is not required in PostgreSQL.
+    [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/49402)
 
-[Tracking Github Issue](https://github.com/cockroachdb/cockroach/issues/43177)
+- The `AddGeometryColumn` spatial function only allows the `true` value for its `use_typmod` parameter.
 
-### Concurrent SQL shells overwrite each other's history
+    [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/49448)
 
-The [built-in SQL shell](cockroach-sql.html) stores its command history in a single file by default (`.cockroachsql_history`). When running multiple instances of the SQL shell on the same machine, therefore, each shell's command history can get overwritten in unexpected ways.
+- CockroachDB does not support the `@` operator for [spatial features](spatial-features.html). Instead of using `@` in spatial expressions, we recommend using the inverse, with `~`. For example, instead of `a @ b`, use `b ~ a`.
```

Comment:
we don't support the `@` operator in general -- see https://github.com/cockroachdb/cockroach/issues/56124.